code/modules/clothing/suits/jobs.dm

```diff
@@ -22,6 +22,14 @@
 	flags_inv = HIDEJUMPSUIT
 	allowed = list(/obj/item/weapon/disk, /obj/item/weapon/stamp, /obj/item/weapon/reagent_containers/food/drinks/flask, /obj/item/weapon/melee, /obj/item/weapon/storage/lockbox/medal, /obj/item/device/assembly/flash/handheld, /obj/item/weapon/storage/box/matches, /obj/item/weapon/lighter, /obj/item/clothing/mask/cigarette, /obj/item/weapon/storage/fancy/cigarettes, /obj/item/weapon/tank/internals/emergency_oxygen)
 
+/obj/item/clothing/suit/capformal

```

Comment:
That just means I have to take away the HIDEJUMPSUIT flag, so I didn't see one way as being any better than the other